### PR TITLE
Add initial translation to Handbook: Preface

### PR DESCRIPTION
**Pull request by yzgyyang**
@ubldr @delphij Please help review. Thanks!

### CODE REVIEW

**Comment by ghost** [comment]
Nice work, it's a good start :)

Please replace the double quotation marks “” (U+201C and U+201D) to ```\"``` within ```<tags>```
example: https://pastebin.com/raw/AAWYriaL

**Comment by yzgyyang** [comment]
@ubldr That's a good catch, it should be fixed by now.

Also, could you please use the built-in code review functionality of GitHub in the future? Just click "Files changed" then leave comments in the code. You can also approve/request changes in this pull request so I can decide to merge the translation. That would be super useful.

Thanks!

File: books/handbook/zh_CN.po [hunk 1]
```diff
@@ -384,7 +384,7 @@ msgstr "<xref linkend=“boot”/> 对第一版内容进行了扩充。"
 #. (itstool) path: listitem/para
 #: book.translate.xml:1032
 msgid "<xref linkend=\"disks\"/> has been written from what used to be two separate chapters on <quote>Disks</quote> and <quote>Backups</quote>. We feel that the topics are easier to comprehend when presented as a single chapter. A section on RAID (both hardware and software) has also been added."
-msgstr "<xref linkend=“disks”/> 由第一版中两个单独的章节“磁盘”和“备份”合并而成。我们认为这两部分作为一个整体比较容易理解。同时 RAID (包括硬件和软件 RAID) 部分也被添加进来。"
+msgstr "<xref linkend=“disks”/> 由第一版中两个单独的章节 <quote>磁盘</quote> 和<quote>备份</quote> 合并而成。我们认为这两部分作为一个整体比较容易理解。同时 RAID (包括硬件和软件 RAID) 部分也被添加进来。"
```
Comment (ghost) [inline books/handbook/zh_CN.po]: Please change ```<xref linkend=“disks”/>``` to ```<xref linkend=\"disks\"/>```

Comment (yzgyyang) [reply books/handbook/zh_CN.po]: Oh! I just noticed that Poedit changes the quote style automatically. Fixed!

**Comment by ghost** [comment]
Thanks for the patch. Looks good to me. :)

**Pull request merged**